src/ir/eh-utils.cpp

```diff
@@ -162,6 +162,34 @@ void handleBlockNestedPops(Function* func, Module& wasm) {
   TypeUpdating::handleNonDefaultableLocals(func, wasm);
 }
 
+Pop* findPop(Expression* expr) {
+  auto pops = findPops(expr);
+  assert(pops.size() == 1);
```

Comment:
Sorry I should've been more clear about this. There can be no pop when a tag doesn't have any params. So we should allow `pops.size()` to either 0 or 1, and should return nullptr in case of 0.

Reply:
Ah, thanks... I should have known that about exception handling.